templates/cortex@.environment.j2

```diff
@@ -0,0 +1,8 @@
+{{ ansible_managed | comment }}
+{# LOOP over environment file to make jaeger envonment file #}
+
+{% for name,value in cortex_env_variables.items() %}
```

Comment:
That does bot seem right, we should only have the variables for the current service